packages/twenty-server/src/engine/metadata-modules/field-metadata/hooks/before-update-one-field.hook.ts

```diff
@@ -89,18 +89,6 @@ export class BeforeUpdateOneField<T extends UpdateFieldInput>
         !updatableFields.includes(key) && !overridableFields.includes(key),
     );
 
-    const isUpdatingLabelWhenSynced =
```

Comment:
all this hook logic is in the context of standards objects. we should not check isLabelSyncedWithName for standard objects as we cannot change the name nor override it